guides/common/modules/proc_applying-configurations.adoc

```diff
@@ -13,21 +13,21 @@ If you have changed some systemd service file, you need to notify systemd daemon
 # systemctl daemon-reload
 ----
 
-Note this will still need to reload the service to do the actual change. It is best to restart whole Satellite:
+After this you still need to reload relevant service for the change to take effect. It is best to restart whole {Project}:
 
 [options="nowrap" subs="attributes"]
 ----
 # {foreman-maintain} service restart
 ----
 
-If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml` you need to rerun installer to apply the changes:
+If you have changed some configuration file (e.g. `/etc/foreman-installer/custom-hiera.yaml`) you need to rerun installer to apply the changes:
```

Comment:
We typically avoid (latin) abbreviations like that according to IBM style guide.

```suggestion
If you have changed a configuration file such as `/etc/foreman-installer/custom-hiera.yaml`, rerun installer to apply your changes:
```